test/Performance/setup_env_unix.sh

```diff
@@ -83,6 +83,28 @@ elif [ $PLATFORM = "RedHat7" ]; then
     pip3 install --upgrade pip >> env_setup.log
     pip3 install pyodbc >> env_setup.log
     printf "done\n"
+    
+elif [ $PLATFORM = "SUSE12" ]; then
+    printf "Update..."
+    sudo zypper refresh >> env_setup.log 2>&1
+    printf "done\n"
+    
+    printf "Installing autoconf, gcc, g++, git, zip, libxml, openssl, python3, pip3..."
+    sudo zypper -n install autoconf gcc gcc-c++ libxml2-devel git zip libopenssl-devel python3-devel python3-pip python3-setuptools >> env_setup.log 2>&1
+    printf "done\n"
+    
+    printf "Installing MSODBCSQL..."
+    zypper -n ar https://packages.microsoft.com/config/sles/12/prod.repo 2>&1
+    zypper --gpg-auto-import-keys refresh 2>&1
+    ACCEPT_EULA=Y zypper -n install msodbcsql >> env_setup.log 2>&1
+    ACCEPT_EULA=Y zypper -n install mssql-tools >> env_setup.log 2>&1
+    zypper -n install unixODBC-devel >> env_setup.log 2>&1
+    printf "done\n"
+    
+    printf "Installing pyodbc"
+    pip3 install --upgrade pip >> env_setup.log 2>&1
+    pip3 install pyodbc >> env_setup.log 2>&1
```

Comment:
Hmm, it's done in a few places kind of haphazardly. I'll make it all consistent.